transport.go

```diff
@@ -0,0 +1,63 @@
+package goftp
+
+import (
+	"crypto/tls"
+	"io"
+	"net/http"
+	"strings"
+)
+
+// Transport implements the http.RoundTripper interface.
+// Typical usage would be to register a Transport to handle
+// ftp:// and/or ftps:// URLs with http.Transport.RegisterProtocol.
+type Transport struct {
```

Comment:
Can we name this `HTTPTransport` so it isn't confused for an ftp transport?

Reply:
I think bare `Transport` works better because it *is* an FTP transport that happens to implement the `http.RoundTripper` interface.

Reply:
Eliminated the separate `Transport` struct in favor of a method on `Config`.